### ISSUE TITLE
How to save the trained model and use for final test like demo?

### ISSUE THREAD

**Issue by DL8614** [muhanzhang/pytorch_DGCNN#28]
Hi,
I am training the model with my own dataset and I am saving the model with
[torch.save(classifier.state_dict(), PATH)](url)

However, when  I load it, it shows [IncompatibleKeys(missing_keys=[], unexpected_keys=[])](url)
It looks that, it saves nothing? How can I save it?

One other question is that. if validation loss looks very low. Then how can I use that trained model with some unknown graph for graph class prediction?

**Comment by muhanzhang** [comment]
It looks like it is saving the model, with no keys missing or unexpected. Maybe this [post](https://discuss.pytorch.org/t/cant-load-state-dict/46041) helps you?

For your second question, do you mean validation accuracy (instead of loss) very low? If so, it is better to tune the architecture/hyperparameters to make it high before applying to unknown graphs.
